Source/DataSources/EntityCluster.js

```diff
@@ -87,6 +87,8 @@ define([
         this._clusterLabelCollection = undefined;
         this._clusterPointCollection = undefined;
 
+        this._collectionIndicesByEntity = Object.create(null);
```

Comment:
`Object.create` is actually a performance bottleneck so as a general rule we don't use it in our code.  Just use `{}` instead.